greenCan/accounts/tests/test_urls.py

```diff
@@ -74,3 +76,18 @@ def test_forget_password_url(self):
     #     self.assertEqual(resolved.func, signup_page)
     #     self.assertEqual(resolved.namespace, "accounts")
     #     self.assertEqual(url, "/accounts/create/")
+
+    # testing profile urls
```

Comment:
I think some code commenting over here could help in understanding the code, besides that its 10/10